src/app/components/JumpTo/index.tsx

```diff
@@ -0,0 +1,74 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import Text from '#app/components/Text';
+import InlineLink from '#app/components/InlineLink';
+
+interface JumpToHeading {
+  id: string;
+  title: string;
+}
+
+interface JumpToProps {
+  jumpToData: {
+    model: {
+      jumpToHeadings: Array<{ heading: string }>;
+    };
+  };
+  eventTrackingData?: EventTrackingMetadata;
+}
+
+const JumpTo = ({ jumpToData, eventTrackingData }: JumpToProps) => {
+  const { translations } = useContext(ServiceContext);
+  const { jumpTo = 'Jump to' } = translations?.articlePage || {};
+
+  const viewRef = useViewTracker(eventTrackingData);
+  const clickTrackerHandler = useClickTrackerHandler({
+    ...eventTrackingData,
+    identifier: 'JumpTo',
```

Comment:
This should be called `componentName` instead of `identifier`.

I'd double check with Product to see the correct format for the value `JumpTo`. It may need to be `jumpto` or `jump-to`.

Reply:
According to the ticket it's `jumpto` - updated both!